tests/analytics/test_analytics.py

```diff
@@ -79,6 +79,10 @@ def test_event_identifiers(analytics_config):
         assert "organization_id" not in properties
 
 
+@pytest.mark.xfail(
```

Comment:
This test was only passing because other tests ran first setting up the context. This test always fails when run in isolation.